docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket.rst

```diff
@@ -7,19 +7,42 @@ Create a dataset from an AWS S3 bucket
 Lightly allows you to configure a remote datasource like Amazon S3 (Amazon Simple Storage Service).
 In this guide, we will show you how to setup your S3 bucket, configure your dataset to use said bucket, and only upload metadata to Lightly.
 
-Lightly needs at minimum to be able to read and list permissions on your bucket. It needs them to provide the Lightly Compute Worker access to your dataset,
+List, Read and Write permissions
+--------------------------------
+
+Lightly needs at minimum to have read and list permissions (s3:GetObject and s3:ListBucket) on your bucket. It needs them to provide the Lightly Worker access to your dataset,
 so that it can process it. Furthermore, the Lightly platform needs access to show you your images in the webapp.
 
-If you want Lightly to create thumbnails for you while uploading the metadata of your images or to write, write permissions are also needed.
+There are 3 reasons which make also write permissions (s3:PutObject) necessary:
+
+- You process videos.
+- You run Lightly on object level.
+- You want Lightly to create thumbnails for you to increase the performance of the WebApp.
```

Comment:
not sure to mention it this way. But any way;
- Webapp -> Lightly platform
- Lightly -> Lightly Worker